event-streams/jmxtrans/README.md

```diff
@@ -0,0 +1,180 @@
+# JmxTrans
+
+[JMX Trans](https://github.com/jmxtrans/jmxtrans) is a tool for collecting data from JMX endpoints of Java applications and sending it to other applications and services. 
+
+The ability to deploy JmxTrans was deprecated in Event Streams 11.1.5 and removed in Event Streams 11.2.0. While the JmxTrans tool remains unsupported, this document provides high-level guidance on how to deploy it separately alongside an Event Streams instance.
+
+## JmxTrans Configuration
+
+[JmxTrans documentation](https://github.com/jmxtrans/jmxtrans/wiki) provides further guidance on defining [queries](https://github.com/jmxtrans/jmxtrans/wiki/Queries) and [output writers](https://github.com/jmxtrans/jmxtrans/wiki/OutputWriters). The configuration depends largely on what information is to be captured and where it must be sent. 
```

Comment:
Could you please explain, how we are achieving this? 

We have removed JmxTrans, how could someone do it now actually? Is that why we are using ConfigMap for? 

What is the role of queries and output writers in this? 

You can use the ConfigMap to configure the JmxTrans, which can be deployed in Event Streams instances that does not support the JmxTrans.